returnn/torch/data/pipeline.py

```diff
@@ -215,11 +218,11 @@ def __init__(self, dataset: torch.utils.data.IterableDataset, batch_size=1, max_
             in one batch (padding included).
             If given as a dict data_key -> value, sets different individual limits per data key.
             If None, no limit.
-            Can also be a callable with kwargs epoch, seq_idx, epoch_continuous, **_other_kwargs,
+            Can also be a callable with kwargs epoch, seq_idx, complete_frac, **_other_kwargs,
```

Comment:
I don't understand?

Reply:
Ah, I mistakenly must have search+replaced this.